lib/osso-abook-contact-field.c

```diff
@@ -4715,11 +4737,34 @@ osso_abook_contact_action_start_with_callback(
       if (contact_action_mmicode(data, values->data))
         return TRUE;
 
-      rv = request_channel(account, OSSO_ABOOK_CONTACT_ACTION_TEL,
+      if (TP_IS_ACCOUNT(account))
```

Comment:
I think a better code would be:

if (TP_IS_ACCOUNT(account))
         rv = request_channel(account, OSSO_ABOOK_CONTACT_ACTION_TEL, values->data, &error);

if (!rv)
        rv = contact_action_tel_as_uri(parent, values->data);